### PR TITLE
Rezk Completion by Yoneda Embedding

### PR DESCRIPTION
**Pull request by kangrongji**
Using the essential image of Yoneda embedding to construct the Rezk completion. Very short and simple thanks to the previous PRs. The drawback is, this construction increases the universe level.

There is another construction using HIT which keeps the universe level. I will give it later.

### CODE REVIEW

**Comment by mortberg** [comment]
Why is the diff still showing things from the other PR?

**Comment by kangrongji** [comment]
> Why is the diff still showing things from the other PR?

It's OK now. Maybe diff can be shown correctly only when master branch is merged.

File: Cubical/Categories/RezkCompletion/Construction.agda [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+open import Cubical.Categories.RezkCompletion.Base
+
+private
+  variable
+    ℓ ℓ' : Level
+
+open isWeakEquivalence
+
+
+-- There are 2 different ways to construct Rezk completion,
```
Comment (mortberg) [inline Cubical/Categories/RezkCompletion/Construction.agda:28]:
```suggestion
-- There are two different ways to construct the Rezk completion,
```

File: Cubical/Categories/RezkCompletion/Construction.agda [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+-- one is using the essential image of Yoneda embbeding,
```
Comment (mortberg) [inline Cubical/Categories/RezkCompletion/Construction.agda:29]:
```suggestion
-- one is using the essential image of the Yoneda embbeding,
```

File: Cubical/Categories/RezkCompletion/Construction.agda [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+-- another one is using higher inductive type
```
Comment (mortberg) [inline Cubical/Categories/RezkCompletion/Construction.agda:30]:
```suggestion
-- another one is using a higher inductive type
```

File: Cubical/Categories/RezkCompletion/Construction.agda [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+-- (c.f. HoTT Book Chaper 9.9).
+
+-- Yoneda embbeding can give a very simple and quick construction.
+-- Unfortunately, this construction increases universe level.
```
Comment (mortberg) [inline Cubical/Categories/RezkCompletion/Construction.agda:34]:
```suggestion
-- Unfortunately, this construction increases the universe level.
```

File: Cubical/Categories/RezkCompletion/Construction.agda [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+
+-- HIT construction, on the other hand, keeps the universe level,
```
Comment (mortberg) [inline Cubical/Categories/RezkCompletion/Construction.agda:36]:
```suggestion
-- The HIT construction, on the other hand, keeps the universe level,
```

**Review by mortberg — CHANGES_REQUESTED** [review]
Just nitpicking on some English in comments. Excellent and simple code!

**Review by mortberg — APPROVED** [review]

**Comment by mortberg** [comment]
Great! Merging once the CI is happy

**Pull request merged**